services/ses/index.js

```diff
@@ -17,6 +17,11 @@ function filterInternalErrors(error) {
   throw error;
 }
 
+function validateContact(contact) {
+  const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/; // eslint-disable-line max-len
```

Comment:
Give up trying to validate an email based on format. The standard is mad and you'll disqualify some valid ones no matter what you do. "Looks like an email" is the best you can do. (@lpil's original suggestion)